src/main/groovy/org/cadixdev/gradle/licenser/LicenseExtension.groovy

```diff
@@ -24,55 +24,56 @@
 
 package org.cadixdev.gradle.licenser
 
+import groovy.transform.PackageScope
 import org.cadixdev.gradle.licenser.header.HeaderFormatRegistry
+import org.gradle.api.Action
 import org.gradle.api.Incubating
 import org.gradle.api.NamedDomainObjectContainer
 import org.gradle.api.Project
-import org.gradle.api.tasks.SourceSet
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.plugins.ExtraPropertiesExtension
+import org.gradle.api.provider.ListProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.provider.ProviderFactory
+import org.gradle.api.resources.TextResourceFactory
 import org.gradle.api.tasks.util.PatternFilterable
 import org.gradle.api.tasks.util.PatternSet
 import org.gradle.util.ConfigureUtil
 
+import javax.inject.Inject
+
 /**
  * Represents the Gradle extension for configuring the settings for the
  * {@link Licenser} plugin.
  */
 class LicenseExtension extends LicenseProperties {
 
-    /**
-     * The source sets to scan for files with license headers.
-     * By default this includes all source sets of the project.
-     */
-    Collection<SourceSet> sourceSets
-
-    /**
-     * The Android source sets to scan for files with license headers.
-     * By default this includes all source sets of the project.
-     *
-     * <p><b>Note:</b> This is only used when the Android Gradle plugin is applied.</p>
-     */
-    @Incubating
-    Collection androidSourceSets
-
     /**
      * The charset to read/write the files with.
      * By default this is {@code UTF-8}.
      */
-    String charset = 'UTF-8'
+    final Property<String> charset
```

Comment:
Isn't this already inherited from `LicenseProperties`?

Reply:
This is -- the override appears to be for providing extra Groovydoc.